ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh

```diff
@@ -105,9 +105,15 @@ fi
 echo 'export KUBECONFIG=/root/dev-scripts/ocp/ostest/auth/kubeconfig' >> /root/.bashrc
 
 timeout -s 9 105m make
-# Sleep for 10 minutes to allow the kube-apiserver to settle down
-# Replace with a proper fix once we establish it helps or not
-sleep 10m
+
+export KUBECONFIG=/root/dev-scripts/ocp/ostest/auth/kubeconfig
```

Comment:
If we want to start aligning all the waiting conditions implemented in `openshift-e2e-test` then we should move this block directly in the baremetalds-e2e-test - now it will work thanks to the proxy

Reply:
Added a PR for that: https://github.com/openshift/release/pull/23258

Reply:
> Added a PR for that: #23258

I could have done it here, anyways want to remove the sleep in your patch also ?

Reply:
As you prefer (in #23258 I added all the waiting conditions from openshift-e2e-test), if you want to move it here and remove the sleep (which is a good idea)  I'll close mine.